Backpack/src/main/java/net/skyscanner/backpack/text/BpkText.kt

```diff
@@ -98,16 +102,41 @@ open class BpkText(
       weight = Weight.values()[weightArg]
     }
 
+    // Adding tint and compoundDrawables does not work. Converting compoundDrawables to compoundDrawablesRelative
+    if (this.layoutDirection == View.LAYOUT_DIRECTION_LTR) {
+      this.setCompoundDrawablesRelative(
+        compoundDrawablesRelative[0] ?: compoundDrawables[0],
+        compoundDrawablesRelative[1] ?: compoundDrawables[1],
+        compoundDrawablesRelative[2] ?: compoundDrawables[2],
+        compoundDrawablesRelative[3] ?: compoundDrawables[3])
+    } else {
+      this.setCompoundDrawablesRelative(
+        compoundDrawablesRelative[0] ?: compoundDrawables[2],
```

Comment:
for `compoundDrawablesRelative` the order is `start`,`top`,`end`,`bottom` while for `compoundDrawables` its `left`,`top`,`right`,`bottom` 
This code fixes the `rtl` even if the developer is using `left`/`right` instead of `start`/`end`

Reply:
Can we add an inline comment in the code explaining this? Is quite confusing right now.

Reply:
Another option instead of inline comment would be to extract constants for `START`, `END`, `LEFT`, `RIGHT`, `TOP` and `BOTTOM`

Reply:
I have tried to make it more readable now. What are your thoughts?